ygot/render.go

```diff
@@ -283,6 +455,40 @@ func leavesToNotifications(leaves map[*path]interface{}, ts int64, prefix []inte
 	return []*gnmipb.Notification{n}, nil
 }
 
+// stripPrefix removes the prefix pfx from the supplied path, and returns the more
+// specific path elements of the path. If the prefix is invalid for the path an
+// error is return.
```

Comment:
It returns an error if the paths are invalid, their types are different, or the prefix does not match the path.